circle.yml

```diff
@@ -5,9 +5,18 @@ machine:
 dependencies:
   override:
     - mvn -U dependency:resolve dependency:resolve-plugins
+
 test:
   override:
     - mvn verify
+  post:
+    - mkdir -p $CIRCLE_TEST_REPORTS/junit/
+    - find . -type f -regex ".*/target/.*-reports/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
+    - cp -r target/coverage-reports/jacoco/ $CIRCLE_ARTIFACTS
+    - curl http://www.jpm4j.org/install/script > jpmInstall.sh
```

Comment:
I'm just wondering whether we could install `jpm` and `codacy-coverage-reporter` in the dependency phase and change the caching settings in a way, that both will be cached.